app/models/room.rb

```diff
@@ -42,7 +43,16 @@ class Room < ApplicationRecord
   scope :all_not_current_room, (lambda do |current_room|
     where("rooms.id != #{current_room}")
   end)
```

Comment:
```suggestion
  scope :all_not_current_room, ->(current_room){ where.not id: current_room }
```
suggest theo ý của em thôi ạ :D

Reply:
ừa làm theo ánh đi, nó hỗ trợ where not mà